test/http-client-base.spec.ts

```diff
@@ -3417,6 +3417,33 @@ describe("http-client-base test", () => {
     expect(response["statusCode"]).to.equal(1000);
     expect(response["responseData"]["isApproved"]).to.equal(true);
   });
+
+  it("query user-request api test", async () => {
+    const testRequestSessionToken = "J4EDHA_oyCyXrtREGS4MpyoGeus";
+
+    const receivedData = {
+      responseTime: 1585467711877,
+      statusCode: 1002,
+      statusMessage: "Accepted",
```

Comment:
Could you change this to `1000` and `Success`? IMO, this API doesn't return status code 202.